### PR TITLE
docs: New Chinese Readme

### PR DESCRIPTION
**Pull request by ShaofeiZi**
Add Chinese instructions to facilitate recommendation to colleagues.

### CODE REVIEW

**Comment by Zizzamia** [comment]
That's dope! I will review this PR ASAP.

File: README-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+[English](./README.md) | 简体中文
+## Why Perfume.js?
+
+- ⏰ 用最新的API获取精准的性能信息
+- 🔨 跨浏览器测试
+- 🚿 过滤虚假的结果
+- 🔭 浏览器跟踪器内置
+- 🤙 支持 async/await 语法
+- 🛰 灵活的跟踪工具
+- ⚡️ 内置策略，不浪费 0 毫秒  [Idle Until Urgent](https://philipwalton.com/articles/idle-until-urgent/)
```
Comment (veronicaz41) [inline README-zh_CN.md]: 内置[Idle Until Urgent](https://philipwalton.com/articles/idle-until-urgent/)策略，0 毫秒浪费

File: README-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+const perfume = new Perfume({
+  firstContentfulPaint: true
+});
+// Perfume.js: First Contentful Paint 2029.00 ms
+```
+
+### 首次输入延迟 (FID)
+
+**FID** 测量用户首次与站点交互时（即，当他们单击链接，点击按钮或使用自定义的，由JavaScript驱动的控件）到浏览器实际能够的时间回应这种互动的延时。
+
```
Comment (veronicaz41) [inline README-zh_CN.md]: **FID** 测量用户首次与站点交互时（即，当他们单击链接，点击按钮或使用自定义的，由JavaScript驱动的控件）到浏览器实际能够回应这种互动的延时。

File: README-zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+  }
+}
+```
+
+## 分析
+
+### Google Analytics
+
+To enable Perfume to send your measures to Google Analytics User timing, set the option `enable:true` and a custom [user timing variable](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#timingVar) `timingVar:"name"`.
+
```
Comment (veronicaz41) [inline README-zh_CN.md]: the translation here is missing:
如果想使用Perfume将你的测量结果传递给Google Analytics User timing，请设置选项 `enable:true` 以及自定义的[user timing variable](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#timingVar) `timingVar:"name"`。

File: README-zh_CN.md [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+// Aync Values
+const durationFCP = await perfume.observeFirstContentfulPaint;
+const durationFID = await perfume.observeFirstInputDelay;
+
+// 将自定义用户时间标识发送到Google Analytics
+perfume.sendTiming(metricName, durationFCP);
+```
+
+## Develop
```
Comment (veronicaz41) [inline README-zh_CN.md]: translation:
开发

**Comment by ShaofeiZi** [comment]
> Very good translations!
> Added a few very minor comments. Can totally merge without addressing these.

Completed the correction of the corresponding text

**Comment by Zizzamia** [comment]
Thank you @ShaofeiZi for this PR and @veronicaz41 for reviewing it.
I'm delighted to see the web community helping each other through Open Source contributions.
🌕🌲🦕

**Pull request merged**